src/modules/tests/candidate/candidate.actions.ts

```diff
@@ -0,0 +1,11 @@
+import { Action } from '@ngrx/store';
+
+export const POPULATE_CANDIDATE_DETAILS = '[JournalEffects] Populate Candidate Details';
+
+export class PopulateCandidateDetails implements Action {
+  readonly type = POPULATE_CANDIDATE_DETAILS;
+  constructor(public payload: any) {}
```

Comment:
Can this be type Candidate instead of any

Reply:
cool